packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -1376,7 +1376,7 @@ function collectRecommendations({
   declaredDependencies.forEach(({key}) => {
     // Does this declared dep satisfy a real need?
     if (satisfyingDependencies.has(key)) {
-      if (suggestedDependencies.indexOf(key) === -1) {
+      if (!suggestedDependencies.includes(key)) {
```

Comment:
[`Array.prototype.includes()` is not supported in IE.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)